packages/edit-post/src/index.js

```diff
@@ -49,6 +49,7 @@ export function initializeEditor(
 	settings,
 	initialEdits
 ) {
+	const isMediumOrBigger = window.matchMedia( '(min-width: 782px)' ).matches;
```

Comment:
Looks like our thresholds are now within the compose package but not expose, we might want to move them somewhere to centralize, not sure where though, so I just did this for now.

Reply:
Probably it's worth adding a small comment that we hide the list view toggle on `medium` viewports which corresponds to this value(782).